# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: toolkit/moz.configure
# Commit: 9973c75413d6
# Full Hash: 9973c75413d6237b9005b9ab908898a153bd2f06
# Author: Paul Adenot <paul@paul.cx>
# Date: 2024-02-28 04:30:28
# Regressor Bug: 1879873
# File Overlap Count: 1
# Description:
#   Bug 1879873 - Remove MOZ_LIBAV_FFT. r=karlt
#   
#   We have some time to use av_fft.h, after which we'll have to use tx.h directly.
#   It's not hard to update.
#   
# ==============================================================================

diff -r 7c947114da80 -r 9973c75413d6 toolkit/moz.configure
--- a/toolkit/moz.configure	Tue Feb 27 12:40:22 2024 +0000
+++ b/toolkit/moz.configure	Tue Feb 27 12:40:22 2024 +0000
@@ -2377,17 +2377,6 @@
 
 # FFmpeg's ffvpx configuration
 # ==============================================================
-with only_when(compile_environment):
-
-    @depends(target)
-    def libav_fft(target):
-        if target.os == "Android" and target.cpu != "arm":
-            return True
-        return target.kernel in ("WINNT", "Darwin") or target.cpu == "x86_64"
-
-    set_config("MOZ_LIBAV_FFT", depends(when=libav_fft)(lambda: True))
-    set_define("MOZ_LIBAV_FFT", depends(when=libav_fft)(lambda: True))
-
 
 # Artifact builds need MOZ_FFVPX defined as if compilation happened.
 with only_when(compile_environment | artifact_builds):
